### PR TITLE
Departure handoff indicator

### LINKED ISSUE #73: Departure Handoff Indicator
With increased capacity for 'non-standard' event splits, I thought it might be a good idea to have a display feature/pop-up that dynamically displays the active handoff frequency.

Some thoughts on functionality:

- Default orders would be defined in the API
- Each SID (or groups of SIDs where relevant?) would be displayed in a box similar in design to the MSL box, with the appropriate frequency
- Hover over frequency to show callsign(?)
- When the frequency changes, it should indicate this with a change of colour
- Area controllers should be able to set non-standard frequencies - then when the plugin detects a change in online controllers, the person that set the non-standard frequency would be prompted to confirm that the non-standard arrangement is still applicable(?)

### PR DESCRIPTION
**Pull request by AndyTWF**
Close #73

### CODE REVIEW

**Comment by AxonC** [comment]
@AndyTWF  can you merge latest dev into this?

**Comment by AndyTWF** [comment]
Conflicts resolved @AxonC :)

File: src/controller/ControllerPositionHierarchy.cpp [hunk 1]
```diff
@@ -40,5 +40,30 @@ namespace UKControllerPlugin {
...
+            if (compare.CountPositions() != this->CountPositions()) {
+                return false;
+            }
+
+            std::vector<std::reference_wrapper<const ControllerPosition>>::const_iterator it1
+                = this->positions.cbegin();
+            std::vector<std::reference_wrapper<const ControllerPosition>>::const_iterator it2
+                = compare.positions.cbegin();
+
+            while (it1 != this->positions.cend())
```
Comment (AxonC) [inline src/controller/ControllerPositionHierarchy.cpp]: This loop could be a little more readable ;)

**Comment by AndyTWF** [comment]
@AxonC Have solved merge conflicts and added a comment to explain, if you wouldn't mind checking?

File: src/controller/ControllerPositionHierarchy.cpp (hunk 1 above)
Comment (AndyTWF) [reply src/controller/ControllerPositionHierarchy.cpp]: Added a comment to make it more meaningful :)

**Pull request merged**